File: C Programming/Assignments/Assignment 4/Q3.c
void main(){
    int r;
    printf("Enter the range from 1 to: ");
    scanf("%d",&r);
    for(int i=1; i<=r; i++){
        int sum = 0;
        for(int j=1; j<i; j++){
            if(i % j == 0){
             sum = sum + j;
            }
        }
    if(sum == i){
        printf("%d ", i);
        }
    }
}

File: Java/C TO JAVA/Assignment 3/Q3.java
//Sum of numbers in given range.
//Find sum of numbers from start to end.
//Input: start = 1, end = 5
//Output: 15//

class Q3
{

public static void main(String args[]){
    int start=1,end=5;
    int tot=0;

    while(start<=end){
    
        tot=start+tot;
        start++;
    }
    System.out.printf("%d",tot);
    
}

    }
